### PR TITLE
Implement a "Recently reviewed features" box on the myfeatures page

### PR DESCRIPTION
**Pull request by jrobbins**
As mentioned at yesterday's team meeting, I realized that it seems like a problem that once a feature is reviewed it just disappears from the pending approvals box.  People like to see their recent progress in a visible way.

In this PR:
* Add a new "Recently reviewed features" box on the myfeatures page.  The accordion for that box is initially closed.
* Implement a new query to get the 10 most recently reviewed features to populate that box.
* Add new unit tests and improve some existing ones.

### CODE REVIEW

File: internals/models_test.py [hunk 1]
```diff
@@ -339,10 +339,17 @@ def setUp(self):
     self.feature_1.put()
     self.feature_1_id = self.feature_1.key.integer_id()
     self.appr_1 = models.Approval(
-        feature_id=self.feature_1_id, field_id=1, state=2,
-        set_on=datetime.datetime.now(),
+        feature_id=self.feature_1_id, field_id=1,
+        state=models.Approval.REVIEW_REQUESTED,
+        set_on=datetime.datetime.now() - datetime.timedelta(1),
```
Comment (KyleJu) [inline internals/models_test.py:344]: Not sure what kinds of clocks it uses for testing, but hope `datetime.datetime.now() - datetime.timedelta(1)` doesn't go below 0?

File: internals/search.py [hunk 2]
```diff
@@ -80,6 +99,8 @@ def process_query(user_query):
     return process_starred_me_query()
   if user_query == 'owner:me':
     return process_owner_me_query()
+  if user_query == 'is:recently-reviewed':
```
Comment (KyleJu) [inline internals/search.py:102]: Are these queries documented anywhere or it is set by the client side?

File: static/elements/chromedash-myfeatures.js [hunk 3]
```diff
@@ -78,9 +78,12 @@ class ChromedashMyFeatures extends LitElement {
...
   }
 
-  renderPendingApprovals() {
-    return this.renderBox(
+  renderPendingAndRecentApprovals() {
+    const pendingBox = this.renderBox(
       'Features pending my approval', 'pending-approval-by:me', 'approvals');
+    const recentBox = this.renderBox(
+      'Recently reviewed features', 'is:recently-reviewed', 'approvals', false);
+    return [pendingBox, recentBox];
```
Comment (KyleJu) [inline static/elements/chromedash-myfeatures.js:86]: What does it mean to return a list of html here? Does it get rendered side by side?

File: internals/models_test.py (hunk 1 above)
Comment (jrobbins) [reply internals/models_test.py:344]: I believe that it uses the regular clock.  It seems to work fine.

File: internals/search.py (hunk 2 above)
Comment (jrobbins) [reply internals/search.py:102]: For now, each query is hard-coded into the client to populate specific boxes in the UI.  

We might generalize this to a user query language at some point in the future.  The fact that these queries are not visible to users means that they do not need end-user documentation and we could rework the query language without users needing to relearn anything.

File: static/elements/chromedash-myfeatures.js (hunk 3 above)
Comment (jrobbins) [reply static/elements/chromedash-myfeatures.js:86]: Not side-by-side, just one string of HTML after the other.   The resulting page looks like this:

image

**Review by KyleJu — APPROVED** [review]

**Pull request merged**